+status check routine

juliyvchirkov opened the pull request:
+status check routine

one can check status of sole memcached daemon if there’re no pools configured (default behaviour if request is made in common way w/o xtra args) or status of some concrete pool from bunch

an outcome is short diagnostic message along w/ appropriate ret code

— status of memcached daemon
/etc/init.d/memcached status
/sbin/service memcached status

— status of memcached pool ’instance001’
/etc/init.d/memcached status instance001
/sbin/service memcached status instance001

— status of memcached pool ’instance014’
/etc/init.d/memcached status instance014
/sbin/service memcached status instance014

dormando commented:
Hi,

Looks like you deleted the repository for this commit? I can't easily retain the authorship without that.

dormando commented:
I'm going to close this, sorry. If you can submit it from a repo that exists I can pull properly and add it for the next round.

thanks!

juliyvchirkov commented:
sorry, your request on the commit slipped my mind in the end of last year while I've been trying to kill two birds with one stone in the end of 2016 to close the overdue tasks before Christmas

yes, I've dropped the copy of memcached repo in my account after this commit 'cause it has been cloned to my account by github automatically & I had no clue that the clone is important and should be retained

I've got here from google to download the init script for memcached w/ multyinstance support, 'cause we've faced the necessity to split the memcached storage for the some projects on our rhel6 server. but also we’ve expected at least from time to time to run an auto health check for the the instances to be sure the caches are alive. so at first when I’ve seen there’s no «status» block in multyinstance memcached init script I’ve been upset for a while & tried to google the update fix. but when I've realized there'll be no solution to download, I've been forced to excogitate some workaround myself 😺

the whole flow from scratch — reviewing the scripts of memcached multyinstance init, finding the possible way to get «status» back, altering the code of init script & final testing & QA — I’ve completed in shell of our centos 6.8  via ssh, & then I've got back here to share the solution. since the job has already been done aside from github, I had no plans to clone the repo, I've expected to  copy & paste my edits on init script & to pull the request directly for this original repo — that’s the reason why I've made no copy of repo preliminary. that’s also the reason why I’ve dropped the auto cloned copy — I've pulled the update right from here & have been surprised when some time later found the clone of memcached w/ my repos; mistakenly assuming that github made a clone for me «just for case» after my edits on original repo, I've droped that copy 

since cloned repo’s been gone, the only proof I’ve been able to find for this moment is the local tweet for my friends https://twitter.com/juliychirkov/status/790718837532622848 w/ offer to review the update I've made 💁🏻

anyway, I’ve cloned the memcached repo  once again & made one more pull request w/ changes https://github.com/memcached/memcached/pull/242

this time it’s even better ’cause last year I’ve forgotten to add the keyword «status» to the [usage section](https://github.com/memcached/memcached/pull/242/files#diff-3329bc6d396055a84fe896b7f424c853R116) 😺

again, sorry for all this mess & inconvenience
